CSID-DGU/2024-2-SCS4031-Happy-7
# Add schedule order functionality and table display

### Changes Introduced
1. Added a new "Schedule Order" section to the dashboard.
   - Includes a form to schedule buy/sell orders with inputs for order type, amount, and target time.
   - Styled for consistency with the existing dashboard.

2. Integrated a dynamic table to display scheduled orders.
   - Fetches data from the server and updates the table every 10 seconds.
   - Displays order time, action (buy/sell), and amount.

3. Implemented JavaScript functionality:
   - Fetches scheduled orders from the API endpoint (`/api/scheduledOrders`).
   - Periodically refreshes the table to reflect updated data.

### How to Test
1. Navigate to the dashboard page.
2. Fill out the "Schedule Order" form and submit a new order.
3. Verify that the new order appears in the table and updates correctly.

### Notes
- Replace `/api/scheduledOrders` with the actual backend API endpoint.
- Styling and layout are aligned with the existing dashboard design.

### Screenshots
[Include screenshots of the new feature if possible]

### Additional Information
- This feature is part of the enhanced user interaction for automated trading management.
